### PR TITLE
Color fixes from GPMDP #130

### PR DESCRIPTION
**Pull request by MarshallOfSound**
- Change the inner color of the pause button to be white
- Set the song indicator (the equalizor animation) to be the white one
- Cluster subtitles are now white

/cc @gmusic-utils/gmusic-js @gmusic-utils/gmusic-theme-js

### CODE REVIEW

File: lib/style.css [hunk 1]
```diff
@@ -101,6 +115,8 @@ body {
 }
 
 .material .song-row .song-indicator {
+  background-image: url('ani_equalizer_white_x2.gif');
```
Comment (MarshallOfSound) [inline lib/style.css]: For anyone wondering about this file path, it is the same as the current one

```
url('ani_equalizer_black_x2.gif');
```

But the color is changed, for some reason google also host  a white one so we can just point to that.  :smile:

**Comment by twolfson** [comment]
Can we get before/after screenshots? It would be nice to see what implications the changes have

File: lib/style.css [hunk 2]
```diff
@@ -56,6 +56,20 @@ body {
...
 
+#player.material paper-icon-button.playing::after {
+  content: '';
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-top: -1.2em;
+  height: 2.4em;
+  width: 2.4em;
```
Comment (twolfson) [inline lib/style.css:67]: Editing height/width seems like it's altering more than the skin. Are we sure we want this?

File: lib/style.css [hunk 3]
```diff
@@ -56,6 +56,20 @@ body {
...
+  background: white;
```
Comment (twolfson) [inline lib/style.css]: White seems very specific, maybe this should be a variable?

File: lib/style.css (hunk 1 above)
Comment (twolfson) [inline lib/style.css]: Maybe we should make this a flag based on "themeType: dark/light" since some people might want a skin?

File: lib/style.css [hunk 4]
```diff
@@ -101,6 +115,8 @@ body {
 }
 
 .material .song-row .song-indicator {
+  background-image: url('ani_equalizer_white_x2.gif');
+  background-size: 28px 28px;
```
Comment (twolfson) [inline lib/style.css]: Any reason for `background-size` here? It seems like the existing indicator should handle that if it is using `background-image`.

File: lib/style.css (hunk 2 above)
Comment (MarshallOfSound) [inline lib/style.css:67]: Should have line noted this aswell, this isn't actually editing the theme.  This is adding the pseudo after element.  Not changing googles theme rather adding a bit to make our theme look right.  I'll post screenshots shortly

File: lib/style.css (hunk 3 above)
Comment (MarshallOfSound) [inline lib/style.css]: Yeah, it should be <<FORE_PRIMARY>> my bad

File: lib/style.css (hunk 4 above)
Comment (MarshallOfSound) [inline lib/style.css]: For some reason the white image is a slightly different scale to the black one, so I needed to make it smaller for it too visually match

File: lib/style.css (hunk 1 above)
Comment (MarshallOfSound) [inline lib/style.css]: There are a few other places in the theme where we force things like `invert(1)`, if we want to make it an option should that be another PR as a feature rather than slotted into this fix PR?

Comment (twolfson) [inline lib/style.css]: Yea, totally. Maybe we should add an inline `TODO` so we don't lose track of them?

**Comment by MarshallOfSound** [comment]
@twolfson Screenshots

## Cluster Titles

#### Before

screen shot 2016-01-29 at 8 00 49 am

#### After

screen shot 2016-01-29 at 8 02 16 am

## Pause Button

#### Before

screen shot 2016-01-29 at 8 01 02 am

#### After

screen shot 2016-01-29 at 8 02 23 am

## Song Indicator

#### Before

screen shot 2016-01-29 at 8 01 39 am

#### After

screen shot 2016-01-29 at 8 02 42 am

**Comment by twolfson** [comment]
Thanks for the screenshots. Nice work on the changes :+1: 

I can't tell if my eyes are playing tricks on me but it looks like the music bars are wider when white o_o

**Comment by MarshallOfSound** [comment]
@twolfson Yeah, I thought that as well.  I think the two images are actually slightly different,  I could make it even smaller?  The dark bars have a default size of `40px 40px` so I'm slightly worried about making it too much smaller, we're already down to like `24px 24px` I think?

File: lib/style.css (hunk 2 above)
Comment (twolfson) [inline lib/style.css:67]: Ah, I see. We are using `z-index: -1` to act as a background. Any reason we don't do a `before`? We shouldn't need the `margin/z-index` adjustments then (`position: absolute` breaks us out of static flow).

Comment (MarshallOfSound) [inline lib/style.css:67]: I'll see if that works, it felt hacky as I did it :P

File: lib/style.css (hunk 1 above)
Comment (MarshallOfSound) [inline lib/style.css]: `TODO:`  added :)

File: lib/style.css (hunk 2 above)
Comment (MarshallOfSound) [inline lib/style.css:67]: I can change to `::before` and can then lose the `z-index` but the margins are still required to centre the fake background.  If you imagine how this works I am putting a white square inside a circle.  The square therefore has to be smaller than the circle so it has to be centred inside the circle :)

Comment (twolfson) [inline lib/style.css:67]: Yea, it totally makes sense. I will leave it to your discretion as to what to do.

Comment (twolfson) [inline lib/style.css:67]: Fwiw, we could grab dimensions via JS but I dislike that as it feels even more like a hack

File: lib/style.css [hunk 5]
```diff
@@ -100,7 +114,13 @@ body {
   padding-top: 0;
 }
 
+/*
+* TODO: Add an option to gmusic-theme.js that handles the difference between
+*       light and dark themes.  The image below should be for "dark" themes only
+*/
 .material .song-row .song-indicator {
+  background-image: url('ani_equalizer_white_x2.gif');
```
Comment (twolfson) [inline lib/style.css]: I was playing around with CSS filters to see if we could get something that works but I couldn't =(

Another option here would be a data URI of our own modified `black_x2.gif`

https://en.wikipedia.org/wiki/Data_URI_scheme

**Comment by twolfson** [comment]
Left this comment in the diff but should have put it in the main thread:

I was playing around with CSS filters to see if we could get something that works but I couldn't =(

Another option here would be a data URI of our own modified `black_x2.gif`

https://en.wikipedia.org/wiki/Data_URI_scheme

**Comment by twolfson** [comment]
The changes look good :+1:

I left something to think about for the image (especially since Google could change their URLs on a whim)

**Comment by MarshallOfSound** [comment]
Yeah, the CSS filters were a no go for me aswell :cry: 

I don't particularly like adding an inline data url to this because that will increase the size tremendously.  You would be inlining a GIF, even single frame PNG's are inlined rarely?

I think I'll go with as is and see how it goes, it's another point of failure for googles GPM changes but you could hope that a URL would be more or less consistent :+1:

**Comment by twolfson** [comment]
Fwiw, the image is pretty small (2kb without gzip) so it's not crazy to inline it. Also, since we have a build step, we could keep it as an actual file in the repo until we compile, then prob make it a variable that we replace like everything else.

**Comment by twolfson** [comment]
Yet another option is to host it in the repo via rawgit and determine the URL via git tag/package.json
